base_layer/core/src/blocks/genesis_block.rs

```diff
@@ -727,24 +723,12 @@ mod test {
             .unwrap();
     }
 
-    fn receiver_commit_nonce_a(commitment: &Commitment) -> PrivateKey {
-        hash_domain!(
-            CommitNonceHashDomain,
-            "com.tari-genesis-tools.multi_party_utxo.receiver_commit_nonce_a",
-            0
-        );
-        let hasher_bytes = DomainSeparatedHasher::<Blake256, CommitNonceHashDomain>::new()
-            .chain(commitment.as_bytes())
-            .finalize();
-        PrivateKey::from_bytes(hasher_bytes.as_ref()).expect("blake256 hash will succeed")
-    }
-
-    // For the faucet, the value of the commitment can be bound into the metadata signature using a deterministic nonce
+    // For the faucet, the value of the commitment can be bound into a value-revealing metadata signature
     fn do_alternate_range_proof_check(factories: &CryptoFactories, outputs: &[&TransactionOutput]) {
```

Comment:
There is a current discussion about adopting these alternate range-proof checks as optional for all outputs in case an optional range proof is omitted, not just for the genesis block faucet outputs. Pending that outcome, I imagine this function or a derivative will be elevated from test code to core code, alongside the optional range-proof implementation.